log/loki.go

```diff
@@ -269,6 +336,34 @@ func (h *lokiHook) loop() {
 	}
 }
 
+func (h *lokiHook) filterLabels(labels map[string]string, msg string) string {
+	if h.allowedLabels == nil {
+		return msg
+	}
+	var b strings.Builder                  // TODO reuse
+	keys := make([]string, 0, len(labels)) // TODO reuse
```

Comment:
One word `TODO`s aren't very helpful if someone other than you will work on them. :)

Can you add more details about why and how these should be reused?

Reply:
I don't want to add `how` as that is likely to change, for example I though of `sync.Pool` when I added in the begining but later decided it will probably better to be a struct field. 

Both of those variables (especially the `strings.Builder`, although I don't know if reusing it will help in performance :shrug: ) will be used all the time in case of a need to filter labels and it's very likely that they will have some similar *bounded* size.

Reply:
Sure, but at least add some more detail than a single word. If you're concerned about performance than mention that as the "why". I also don't think detailing the "how" is helpful as it might distract from the fix, but `TODO`s are easier to address if they're actionable and detailed, as you've explained it here, but maybe more succinctly.

Reply:
fixed in a93d7db